CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.3.3] - 2023-10-04
+
```

Comment:
Curious do we need something like 
```
[Full changelog](https://github.com/openfga/openfga/compare/v1.3.2...v1.3.3)
```
And don't know why we have it for some but not others

Reply:
ah yeah i can do that.